torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -45,12 +45,33 @@ c10::IValue getFunctionTuple(const Function& func) {
 
   // operator names
   std::vector<c10::OperatorName> opnames;
+  std::vector<IValue> fn_names;
   for (size_t i = 0; i < code.instructions().size(); ++i) {
     Instruction ins = code.instructions()[i];
     if (ins.op == OP || ins.op == OPN) {
       auto node = code.instructions_source()[i];
       opnames.emplace_back(node->schema().operator_name());
     }
+    // Emit qualified names for built-in Functions
+    // (i.e. Function instances for which isGraphFunction() returns false)
+    if (ins.op == CALL) {
+      auto node = code.instructions_source()[i];
+      if (node->kind() == prim::CallMethod) {
+        if (auto class_type = node->inputs().at(0)->type()->cast<ClassType>()) {
```

Comment:
I think we should emit an INTERACE_CALL here so that the mobile interpreter can look the builtin function up from the class type rather than have to have a separate table of builtin methods.

Second, I think the CallFunction handling is not going to work if the function is not a builtin but there is no check here to prevent exporting a function call. It either needs to be guarded or we need to make sure that the functions being called are also being exported as bytecode.

Reply:
Third, is the CallFunction pathway here even being taken? fn_names on the importer are only ever used to lookup class methods. So any CallFunction is not going to work.